db/music.sql

```diff
@@ -138,210 +138,211 @@ CREATE TABLE `songs` (
+	(22,'my tears ricochet',3.26,7,0,0,NULL),
+	(23,'mirrorball',3.54,7,0,0,NULL),
+	(24,'seven',3.27,7,0,0,NULL),
+	(25,'august',3.54,7,0,0,NULL),
+	(26,'this is me trying',3.27,7,0,0,NULL),
+	(27,'illicit affairs',3.05,7,0,0,NULL),
+	(28,'invisible string',3.13,7,0,0,NULL),
+	(29,'mad woman',3.16,7,0,0,NULL),
+	(30,'epiphany',3.24,7,0,0,NULL),
+	(31,'betty',3.08,7,0,0,NULL),
+	(32,'peace',3.41,7,0,0,NULL),
+	(33,'hoax',3.15,7,0,0,NULL),
+	(34,'willow',3.57,8,0,0,NULL),
+	(35,'champagne problems',3.05,8,0,0,NULL),
+	(36,'gold rush',3.08,8,0,0,NULL),
+	(37,'\'tis the damn season',3.02,8,0,0,NULL),
+	(38,'tolerate it',3.09,8,0,0,NULL),
+	(39,'no body, no crime (feat. HAIM)',3.20,8,0,0,NULL),
+	(40,'happiness',3.21,8,0,0,NULL),
+	(41,'dorothea',3.26,8,0,0,NULL),
+	(42,'coney island (feat. The National)',3.09,8,0,0,NULL),
+	(43,'ivy',3.42,8,0,0,NULL),
+	(44,'cowboy like me',3.53,8,0,0,NULL),
+	(45,'long story short',3.15,8,0,0,NULL),
+	(46,'marjorie',3.55,8,0,0,NULL),
+	(47,'closure',3.26,8,0,0,NULL),
+	(48,'Here He Comes',3.18,9,0,0,NULL),
+	(49,'Party Starter',3.57,9,0,0,NULL),
+	(50,'Switch',3.14,9,0,0,NULL),
+	(51,'Mr. Niceguy',3.40,9,0,0,NULL),
+	(52,'Ms. Holy Roller',3.29,9,0,0,NULL),
+	(53,'Tell Me Why',3.59,9,0,0,NULL),
+	(54,'I Wish I Made That / Swagga',3.35,9,0,0,NULL),
+	(55,'Shape of You',3.49,10,0,0,NULL),
+	(56,'Castle on the Hill',3.51,10,0,0,NULL),
+	(57,'Galway Girl',3.18,10,0,0,NULL),
+	(58,'One',3.10,11,0,0,NULL),
+	(59,'I\'m a Mess',3.30,11,0,0,NULL),
+	(60,'Sing',3.22,11,0,0,NULL),
+	(61,'Don\'t',3.42,11,0,0,NULL),
+	(62,'Nina',3.56,11,0,0,NULL),
+	(63,'Photograph',3.13,11,0,0,NULL),
+	(64,'Bloodstream',3.26,11,0,0,NULL),
+	(65,'Tenerife Sea',3.58,11,0,0,NULL),
+	(66,'Runaway',3.40,11,0,0,NULL),
+	(67,'The Man',3.04,11,0,0,NULL),
+	(68,'Thinking Out Loud',3.42,11,0,0,NULL),
+	(69,'Afire Love',3.11,11,0,0,NULL),
+	(70,'Hello',3.19,12,0,1,NULL),
+	(71,'When We Were Young',3.50,12,0,1,NULL),
+	(72,'Send My Love (To Your New Lover)',3.41,12,0,0,NULL),
+	(73,'Rolling in the Deep',3.45,13,0,0,NULL),
+	(74,'Someone Like You',3.25,13,0,0,NULL),
+	(75,'Set Fire to the Rain',3.56,13,0,0,NULL),
+	(76,'Chasing Pavements',3.57,14,0,0,NULL),
+	(77,'Hometown Glory',3.42,14,0,0,NULL),
+	(78,'Cold Shoulder',3.36,14,0,0,NULL),
+	(79,'Bodak Yellow',3.22,15,0,0,NULL),
+	(80,'I Like It',3.10,15,0,0,NULL),
+	(81,'Be Careful',3.08,15,0,0,NULL),
+	(82,'Peaches (feat. Daniel Caesar & Giveon)',3.34,16,0,0,NULL),
+	(83,'Hold On',3.48,16,0,0,NULL),
+	(84,'Anyone',3.10,16,0,0,NULL),
+	(85,'Yummy',3.55,17,0,0,NULL),
+	(86,'Intentions (feat. Quavo)',3.54,17,0,0,NULL),
+	(87,'Forever (feat. Post Malone & Clever)',3.46,17,0,0,NULL),
+	(88,'What Do You Mean?',3.14,18,0,0,NULL),
+	(89,'Sorry',3.49,18,0,0,NULL),
+	(90,'Love Yourself',3.04,18,0,0,NULL),
+	(91,'Wannabe',3.33,19,1,0,'2023-05-18 09:43:15'),
+	(92,'Say You\'ll Be There',3.32,19,2,1,'2023-05-18 09:43:19'),
+	(93,'2 Become 1',3.41,19,1,1,'2023-05-18 09:43:18'),
```

Comment:
Set all play counts to '0' and timestamps to 'NULL' in SQL to reset the database fields